src/main/kotlin/com/bardsoftware/backend/fleet/rmarkdown/Subscriber.kt

```diff
@@ -42,25 +43,45 @@ internal class MessageReceiverExample : MessageReceiver {
     }
 }
 
+class SubscribeManager(subscriptionId: String) {
+    private val dockerProcessor = DockerProcessor()
+    private val subscriptionName = SubscriptionName.of(PROJECT_ID, subscriptionId)
+    private var subscriber: Subscriber? = null
+
+    fun subscribe(callback: (message: String, md5sum: String?) -> Unit) {
+        try {
+            subscriber = Subscriber.newBuilder(subscriptionName, MessageReceiverExample()).build()
```

Comment:
declare val outside of try block and make it non-nullable

Reply:
Done